### ISSUE TITLE
devtools.tsx - Cannot read properties of null (reading state)

### ISSUE THREAD

**Issue by xeinebiu** [TanStack/router#815]
### Describe the bug

Ater upgrading from version 194 to 211, the router devtools  does throw an error while totally fine before updating.

![Untitled](https://github.com/TanStack/router/assets/50703441/361aca5c-e6cc-401a-8cd2-1ca0d35d3e0f)

### Your Example Website or App

_blank_

### Steps to Reproduce the Bug or Issue

```tsx
return (
		<>
			<RouterProvider router={router} basepath={basePath} />
			<TanStackRouterDevtools router={router} />
		</>
	);
```

### Expected behavior

The devtools "useRouter" should receive a valid instance of router.

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by shansmith01** [comment]
Can confirm this

**Comment by radoslavkarlik** [comment]
204 is the last working version.

**Comment by maamri95** [comment]
@xeinebiu [Manually passing the Router Instance](https://tanstack.com/router/v1/docs/devtools#manually-passing-the-router-instance) not work from 204 for not have this issue you need [Using inside of the RouterProvider](https://tanstack.com/router/v1/docs/devtools#using-inside-of-the-routerprovider)

**Comment by tannerlinsley** [comment]
We've fixed this by shipping our custom cross package context. Should work fine now. Reopen with a reproduction if you see it again.

**Comment by schiller-manuel** [comment]
> We've fixed this by shipping our custom cross package context. Should work fine now. Reopen with a reproduction if you see it again.

how can we use the devtools based on this new "cross package context"? the docs do not reflect this update yet.

**Comment by tannerlinsley** [comment]
You shouldn’t have to do anything

**Comment by austinm911** [comment]
> You shouldn’t have to do anything
> […](#)
> On Dec 14, 2023 at 7:04 PM -0700, Manuel Schiller ***@***.***>, wrote: > We've fixed this by shipping our custom cross package context. Should work fine now. Reopen with a reproduction if you see it again. how can we use the devtools based on this new "cross package context"? the docs do not reflect this update yet. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you modified the open/close state.Message ID: ***@***.***>

I upgraded from 226 to 273 for the devtool and router packages and have this issue now. Whether i pass it alongside the router or in a child route it doesn't work.
![image](https://github.com/TanStack/router/assets/31991302/1af979b7-bce1-449c-9c81-fe6cb6c6d593)


![image](https://github.com/TanStack/router/assets/31991302/af5cf2b8-e47f-422a-8368-74257339a0ac)

**Comment by igortas** [comment]
Still available in ```"@tanstack/router-devtools": "^1.20.0"```, opened question in ```Q&A``` section...
